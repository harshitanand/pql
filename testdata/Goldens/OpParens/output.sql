SELECT * FROM "Tokens" WHERE ("Value" - 3) > 0 ORDER BY "Value" ASC NULLS FIRST;
