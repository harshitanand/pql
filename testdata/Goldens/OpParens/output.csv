4,TokenString
5,TokenAnd
6,TokenOr
